components/tutorial/tutorial_tip/tutorial_tip.tsx

```diff
@@ -255,20 +212,14 @@ export default class TutorialTip extends React.PureComponent<Props, State> {
         if (this.props.extraFunc) {
             this.props.extraFunc();
         }
-
-        if (onNextNavigateTo && nextStep === true && autoTour) {
```

Comment:
I think this is still needed by the CRT tour:
```
components/crt_tour/crt_list_tutorial_tip/crt_list_tutorial_tip.tsx
components/crt_tour/crt_welcome_tutorial_tip/crt_welcome_tutorial_tip.tsx
```

Reply:
@neallred Yeah right